cropper/src/main/java/com/canhub/cropper/CropImageActivity.kt

```diff
@@ -62,14 +62,19 @@ open class CropImageActivity :
         if (savedInstanceState == null) {
             if (cropImageUri == null || cropImageUri == Uri.EMPTY) {
                 when {
-                    cropImageOptions.imageSourceIncludeGallery &&
-                        cropImageOptions.imageSourceIncludeCamera ->
-                        showImageSourceDialog(::openSource)
-                    cropImageOptions.imageSourceIncludeGallery ->
-                        pickImageGallery.launch("image/*")
-                    cropImageOptions.imageSourceIncludeCamera ->
-                        openCamera()
-                    else -> finish()
+                    cropImageOptions.showIntentChooser -> showIntentChooser()
+                    else -> {
+                        when {
```

Comment:
We don't need two `when` statements here. Could everything be in one. 
```kt
when {
    cropImageOptions.showIntentChooser -> 
        showIntentChooser()
    cropImageOptions.imageSourceIncludeGallery &&
    cropImageOptions.imageSourceIncludeCamera ->
        showImageSourceDialog(::openSource)
    cropImageOptions.imageSourceIncludeGallery ->
        pickImageGallery.launch("image/*")
    cropImageOptions.imageSourceIncludeCamera ->
        openCamera()
    else -> 
        finish()
```

Reply:
Oh right. How did I not realise this?